README.md

```diff
@@ -165,6 +165,10 @@ It scales nicely from just a single server to thousands of servers, even in comp
 
   Check also the [Netdata Deployment Strategies](https://learn.netdata.cloud/docs/architecture/deployment-strategies) to decide how to deploy it in your infrastructure.
 
+  By default, you will have immediately available a local dashboard. Netdata starts a web server for its dashboard at port `19999`. Open up your web browser of choice and
+navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If you're unsure, try
```

Comment:
What does it mean "If you're unsure"? I think we can safely assume that the person who can install Netdata knows the host's IP address (or hostname).

Reply:
@ilyam8 I copied it from [here](https://github.com/netdata/netdata/blob/31924c0eb2fb245bdf047a5f4fe9f5ef10baf24e/web/gui/README.md?plain=1#L16), from what it is to work just a fallback suggestion don't actually see a harm keeping it here

Reply:
I see the harm - the user can open different Netdata (local) after installation on the remote host.

Reply:
My suggestion is to remove the sentence or change the wording - not "if unsure" but if installed on Localhost.

Reply:
got it, changed it to
> If installed on localhost, you can access it through `http://localhost:19999`.